src/exports.jl

```diff
@@ -119,6 +119,8 @@ export
     Field,
     evaluate_at_grid_nodes,
     apply_analytical!,
+    getgrid,
+    getdim,
```

Comment:
Do these have to be exported? In particular `getdim` might be ambiguous. Perhaps if we want to export it should be split into `get_refdim` and `get_spacedim`?

Reply:
Good point. Should I do it in this PR or a separate one?

Reply:
I will make a separate one because I feel like this needs further discussion.